Kickstarter-iOS/ViewModels/AppDelegateViewModelTests.swift

```diff
@@ -484,12 +462,11 @@ final class AppDelegateViewModelTests: TestCase {
 
     XCTAssertEqual(["App Open", "Opened App"], self.trackingClient.events)
 
-    let result = self.vm.inputs.applicationOpenUrl(
```

Comment:
This no longer has a return value because we never used it before, it was always `false`.